tests/test_sentences_endpoints.py

```diff
@@ -0,0 +1,127 @@
+import pytest
+import json
+from .factory import insertDummySentences, removeCollections
+from app import create_app
+from app.models import *
+import sys
+sys.path.append("..")
+
+
+@pytest.yield_fixture
+def app():
+    app = create_app("testing")
+    yield app
+
+
+@pytest.fixture
```

Comment:
It looks like this is going to be in all the classes. You might create a utility for it.

Reply:
This is actually a good suggestion, I never though about that. I will into how to that.